demos/drawer/permanent-drawer-above-toolbar.html

```diff
@@ -144,6 +156,16 @@ <h1 class="mdc-typography--display1">Permanent Drawer</h1>
       document.querySelector('#toggle-tall').addEventListener('click', function() {
         extraTall.style.display = extraTall.style.display ? '' : 'none';
       });
+
+      var drawerEl = document.querySelector('#demo-theme-switch');
```

Comment:
This should probably be `#demo-drawer`

Ditto in other files

Reply:
don't know what I was thinking there...changing